protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchema.java

```diff
@@ -108,6 +108,10 @@
 
   private transient Descriptor descriptor;
 
+  private transient int hashCode = NO_HASHCODE;
+
+  private static final int NO_HASHCODE = Integer.MIN_VALUE;
```

Comment:
Would it be better for this to be 0, to better align with the Java [spec](https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#hashCode(java.lang.Object[])) ?

Reply:
Probably either can be used, I'm copying what Avro does here:  https://github.com/apache/avro/blob/master/lang/java/avro/src/main/java/org/apache/avro/Schema.java#L111